[2022-05-03 06:42:13,036] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: user_process.processing_user 2022-05-02T00:00:00+00:00 [queued]>
[2022-05-03 06:42:13,041] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: user_process.processing_user 2022-05-02T00:00:00+00:00 [queued]>
[2022-05-03 06:42:13,041] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2022-05-03 06:42:13,041] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2022-05-03 06:42:13,041] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2022-05-03 06:42:13,047] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): processing_user> on 2022-05-02T00:00:00+00:00
[2022-05-03 06:42:13,049] {standard_task_runner.py:52} INFO - Started process 2602 to run task
[2022-05-03 06:42:13,052] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'user_process', 'processing_user', '2022-05-02T00:00:00+00:00', '--job-id', '42', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/user_process.py', '--cfg-path', '/tmp/tmp8jtrc5hx', '--error-file', '/tmp/tmpyge_bm77']
[2022-05-03 06:42:13,052] {standard_task_runner.py:77} INFO - Job 42: Subtask processing_user
[2022-05-03 06:42:13,086] {logging_mixin.py:104} INFO - Running <TaskInstance: user_process.processing_user 2022-05-02T00:00:00+00:00 [running]> on host airflowvm
[2022-05-03 06:42:13,114] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=user_process
AIRFLOW_CTX_TASK_ID=processing_user
AIRFLOW_CTX_EXECUTION_DATE=2022-05-02T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-02T00:00:00+00:00
[2022-05-03 06:42:13,122] {python.py:151} INFO - Done. Returned value was: None
[2022-05-03 06:42:13,124] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=user_process, task_id=processing_user, execution_date=20220502T000000, start_date=20220503T064213, end_date=20220503T064213
[2022-05-03 06:42:13,137] {taskinstance.py:1245} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-05-03 06:42:13,168] {local_task_job.py:151} INFO - Task exited with return code 0
